benchmark/transaction-log.bench.ts

```diff
@@ -0,0 +1,114 @@
+import { describe } from 'vitest';
+import { benchmark, concurrent } from './setup.js';
+
+describe('Transaction log', () => {
+	const ENTRY_COUNT = 1000;
+	const data = Buffer.alloc(100, 'a');
+
+	describe('write log with 100 byte records', () => {
+		benchmark('rocksdb', concurrent({
+			async setup(ctx) {
```

Comment:
Need a type to make TS happy.

```suggestion
			async setup(ctx:  BenchmarkContext<RocksDatabase>) {
```